optimum/exporters/tasks.py

```diff
@@ -925,8 +949,14 @@ def get_supported_model_type_for_task(task: str, exporter: str) -> List[str]:
         ]
 
     @staticmethod
-    def format_task(task: str) -> str:
-        return task.replace("-with-past", "")
+    def map_from_legacy(task: str) -> str:
+        if task in TasksManager._LEGACY_TASK_MAP:
+            logger.warning(
+                f"The task name {task} is deprecated, and will not be supported in"
+                f" optimum 2.0. Please use {TasksManager._LEGACY_TASK_MAP[task]} instead."
```

Comment:
We should always accept both IMO.
Basically the previous naming was due because it is aligned with Auto classes for many models. So accepting both makes sense. 
```suggestion
```